src/fparser/common/readfortran.py

```diff
@@ -515,8 +515,26 @@ def __init__(self, prefix, block, suffix, linenospan, reader, message):
         FortranReaderError.__init__(self, message)
 
 
+class CppDirective(Line):
+    '''Holds a preprocessor directive source line.
+
+    :param str line: String containing the text of a single or \
```

Comment:
Could you please start the description using lower case i.e. `String` becomes `string`. This is because of the way descriptions are rendered in Sphinx. Similarly for the other lines. Could you also add a `.` to the end of this line.